sklearn/decomposition/factor_analysis.py

```diff
@@ -347,3 +359,38 @@ def score(self, X, y=None):
             Average log-likelihood of the samples under the current model
         """
         return np.mean(self.score_samples(X))
+
+    def _rotate(self, method="varimax", n_components=None, tol=1e-6):
+        "Rotate the factor analysis solution."
+        implemented = {"varimax", "quartimax"}
+        if method in implemented:
+            self.components_ = self._ortho_rotation(
+                self.components_.T, method=method, eps=tol)[:n_components]
+        else:
+            raise NotImplementedError("'method' must be in %s, not %s"
+                                      % (implemented, method))
+
+    def _ortho_rotation(self, components, method='varimax', eps=1e-6,
+                        itermax=100):
+        """Return rotated components."""
+        if (method == 'varimax'):
```

Comment:
parentheses should be removed here and below.

Can just use `gamma = method == 'varimax'`, even.